tgui/packages/tgui/interfaces/Mecha/ModulesPane.tsx

```diff
@@ -416,33 +422,79 @@ const SnowflakeSleeper = (props) => {
           />
         }
       >
-        {patient.patientname}
+        {patient.patient_name}
       </LabeledList.Item>
       <LabeledList.Item label={'Health'}>
-        {patient.is_dead ? (
-          <Box color="red">Patient dead</Box>
-        ) : (
-          <ProgressBar
-            ranges={{
-              good: [0.75, Infinity],
-              average: [0.25, 0.75],
-              bad: [-Infinity, 0.25],
-            }}
-            value={patient.patient_health}
-          />
-        )}
-      </LabeledList.Item>
-      <LabeledList.Item label={'Detailed Vitals'}>
-        <Button
-          content={'View'}
-          onClick={() =>
-            act('equip_act', {
-              ref: ref,
-              gear_action: 'view_stats',
-            })
-          }
+        <ProgressBar
+          ranges={{
+            good: [0.75, Infinity],
+            average: [0.25, 0.75],
+            bad: [-Infinity, 0.25],
+          }}
+          value={patient.patient_health}
         />
       </LabeledList.Item>
+      <LabeledList.Item className="candystripe" label={'State'}>
```

Comment:
No need to enclose strings in an object like this, just `label="State"`

Reply:
```suggestion
      <LabeledList.Item className="candystripe" label="State">
```